bin/build-verify.mjs

```diff
@@ -45,7 +46,9 @@ for (let filePath of files) {
   let content = file.toString();
 
   for (let searchFor of FORBIDDEN) {
-    if (content.match(searchFor)) {
+    const match = typeof searchFor === 'string' ? content === searchFor : searchFor.test(content);
```

Comment:
Bug (?) caught by linting. Even if it somehow works, it seems good to make the code match the types.